src/Containers/Microsoft.NET.Build.Containers/Tasks/CreateNewImage.cs

```diff
@@ -99,6 +103,12 @@ internal async Task<bool> ExecuteAsync(CancellationToken cancellationToken)
                 Log.LogErrorFromException(e, showStackTrace: false, showDetail: true, file: null);
                 return !Log.HasLoggedErrors;
             }
+            catch (BaseImageNotFoundException e)
+            {
+                telemetry.LogRidMismatch(e.RequestedRuntimeIdentifier, e.AvailableRuntimeIdentifiers.ToArray());
```

Comment:
Is a base image not being found equivalent to a RID mismatch error? It looks like it, based on the above.

Reply:
Yes, for our specific use case - the image _must_ have architecture information on it per spec, and we force alignment between the RID we target and the architecture information on the image.